ts/src/pro/hyperliquid.ts

```diff
@@ -413,7 +413,7 @@ export default class hyperliquid extends hyperliquidRest {
         }
         if (!(timeframe in this.ohlcvs[symbol])) {
             const limit = this.safeInteger (this.options, 'OHLCVLimit', 1000);
-            const stored = new ArrayCacheByTimestamp (limit);
+            const stored = new ArrayCacheByTimestamp<OHLCV> (limit);
```

Comment:
@pcriadoperez Question, don't we need to update every place where we initialize these structures?

Reply:
It is not needed as it defaults to unknown, but I'll add it as it's good practice. I also added that you can only add an item of the same type as the array